llvm/test/tools/sycl-post-link/auto-module-split-1.ll

```diff
@@ -10,27 +10,27 @@ target triple = "spir64-unknown-linux"
 
 $_Z3barIiET_S0_ = comdat any
 
-; CHECK-TU0-NOT: @{{.*}}GV{{.*}}
-; CHECK-TU1: @{{.*}}GV{{.*}} = internal addrspace(1) constant [1 x i32] [i32 42], align 4
+; CHECK-TU1-NOT: @{{.*}}GV{{.*}}
```

Comment:
all test changes are related to re-numeration of output files - E.g. ESIMD kernels appeared in files generated earlier (with smaller numeric index), now they will appear in files generated later, etc.